addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -388,11 +389,20 @@ export class KanbanRenderer extends Component {
                         context,
                         title: this.env._t("Create"),
                         onRecordSaved: (record) => {
-                            group.addRecord(record, 0);
+                            prom = group.createRecord(
```

Comment:
isn't it enough to await here? It seems that the FormViewDialog waits for the promise before closing the dialog

Reply:
ok right